src/amr/solvers/solver_ppc.hpp

```diff
@@ -94,6 +94,7 @@ class SolverPPC : public ISolver<AMR_Types>
                               double const currentTime, double const newTime) override;
 
 
+    void regrid() override { ionUpdater_.reset(); }
```

Comment:
name `regrid` may be misleading as the operation does not regrid per se. I suggest `applyOnRegrid()` maybe, or `onRegrid()`